test/built-ins/Proxy/apply/trap-is-undefined.js

```diff
@@ -2,20 +2,30 @@
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
 es6id: 9.5.13
+esid: sec-proxy-object-internal-methods-and-internal-slots-call-thisargument-argumentslist
 description: >
-    If trap is undefined, propagate the call to the target object.
+    If trap is null or undefined, propagate the call to the target object.
 info: >
     [[Call]] (thisArgument, argumentsList)
 
     7. If trap is undefined, then Return Call(target, thisArgument,
     argumentsList).
 ---*/
 
-var target = function(a, b) {
+let ctx = {};
+let target = function(a, b) {
+    assert.sameValue(this, ctx);
     return a + b;
 };
-var p = new Proxy(target, {
-    apply: undefined
-});
 
-assert.sameValue(p(1, 2), 3);
+let handlers = [
+    {},
+    {apply: undefined},
+    {apply: null},
+];
+
+for (let handler of handlers) {
+    let p = new Proxy(target, handler);
+    let res = Reflect.apply(p, ctx, [1, 2]);
+    assert.sameValue(res, 3);
```

Comment:
I would rather have each test separated. If not on the same file, at least on different lines, not in a loop. DRY does not help much here, and an assertion failure can't tell what part is failing.